alicloud/table_alicloud_cs_kubernetes_cluster.go

```diff
@@ -461,9 +510,9 @@ func getCsKubernetesClusterAka(ctx context.Context, d *plugin.QueryData, h *plug
 	commonColumnData := commonData.(*alicloudCommonColumnData)
 	accountID := commonColumnData.AccountID
 
-	akas := []string{"acs:cs:" + data["region_id"].(string) + ":" + accountID + ":container/" + data["name"].(string)}
+	arn := "acs:cs:" + data["region_id"].(string) + ":" + accountID + ":container/" + data["name"].(string)
```

Comment:
@ParthaI Can we update the `arn` to `arn:acs:cs:cn-hangzhou:123456789xxxxxxx:cluster/cc5b5364662424e3ba2xxxx`
https://www.alibabacloud.com/help/doc-detail/172061.htm#title-9h3-co9-czq

Reply:
Updated the arn format as per requirement